proposed/0007-Address_relocation.md

```diff
@@ -0,0 +1,40 @@
+- Feature Name: Prevent any single address being used more than once during address relocation.
```

Comment:
The file name doesn't need to contain a prospective RFC number.
Please DO NOT commit a change to fix it before all the other comments got resolved, otherwise the comment history will be lost